oneview_redfish_toolkit/app.py

```diff
@@ -315,10 +315,11 @@ def hp_oneview_client_exception(exception):
 
     if config.auth_mode_is_conf():
         # Loading scmb connection
-        if scmb.check_cert_exist():
-            logging.info('SCMB certs already exists testing connection...')
+        if scmb.has_valid_certificates():
+            logging.info('SCMB certs already exists and is valid...')
         else:
-            logging.info('SCMB certs not found. Generating/getting certs....')
+            logging.info('SCMB certs not found. '
+                         'Checking if it was already generated in Oneview...')
```

Comment:
We could place all this **if config.auth_mode_is_conf():** code block as a new function **init_event_service** inside scmb.py file. This way we could create new unit tests for this code block,  since we do not test the app.py